charts/prometheus-operator-admission-webhook/Chart.yaml

```
@@ -2,7 +2,7 @@
 apiVersion: v2
 description: Prometheus Operator Admission Webhook
 name: prometheus-operator-admission-webhook
-version: 0.24.2
+version: 0.24.3
```

Should really be a minor change.
```suggestion
version: 0.25.0
```